bwt/bitvector_test.go

```diff
@@ -0,0 +1,112 @@
+package bwt
+
+import (
+	"testing"
+)
+
+type GetBitTestCase struct {
```

Comment:
should this be exposed?

Reply:
It Shouldn't be. Definitions in _test aren't public. 

```go
package poly

import (
	"fmt"

	"github.com/bebop/poly/bwt"
)

func polyRoot() {
	thing := bwt.GetBitTestCase{}
	fmt.Println(thing)
}

```

^ That won't build